test/selenium_start.sh

```diff
@@ -28,7 +28,15 @@ systemctl start docker
 
 docker run -d -p 4444:4444 --name selenium-hub selenium/hub:3
 wait_curl /grid/console "Grid Console"
-docker run -d --link selenium-hub:hub --shm-size=512M selenium/node-chrome:3
-wait_curl /grid/console "browserName: chrome"
-docker run -d --link selenium-hub:hub --shm-size=512M selenium/node-firefox:3
-wait_curl /grid/console "browserName: firefox"
\ No newline at end of file
+
+if [ -z ${DEBUG_TEST+x} ]; then
+    docker run -d --link selenium-hub:hub --shm-size=512M selenium/node-chrome:3
+    wait_curl /grid/console "browserName: chrome"
+    docker run -d --link selenium-hub:hub --shm-size=512M selenium/node-firefox:3
+    wait_curl /grid/console "browserName: firefox"
+else
+    docker run -d -P -p 5901:5900 --link selenium-hub:hub -e VNC_NO_PASSWORD=1 --shm-size=512M selenium/node-chrome-debug:3
+    wait_curl /grid/console "browserName: chrome"
+    docker run -d -P -p 5902:5900 --link selenium-hub:hub -e VNC_NO_PASSWORD=1 --shm-size=512M selenium/node-firefox-debug:3
+    wait_curl /grid/console "browserName: firefox"
+fi
```

Comment:
Is there any downside with always running the -debug variants? This of course requires changing the containers on Cockpit's selenium image, but that's fine -- we would need to put them on the image for this option anyway. So instead of having two, I'd much rather continue to only have one.